src/components/Header.js

```diff
@@ -1,8 +1,25 @@
-import React from 'react';
+import React, {useState, useEffect} from 'react';
 import { Outlet, Link } from "react-router-dom";
+import axios from 'axios';
+
 const Header = () => {
+    const [authoName, setAuthorName] = useState('');
+    const [imageLoc, setImageLoc] = useState('');
+
+    const loadAuthor = async () => {
+        await axios.get('https://api.github.com/users/vuejs', {
+        }).then(response => {
+            setAuthorName(response.data.login)  
+            setImageLoc(response.data.avatar_url)
+        }).catch(function(error) {
+            console.error(error);
+        });
+     }
```

Comment:
Remove the async await since you using promise.
You can just use one, either async await or promise.
Also apply fix to other related files